style/icons/MUTCD/I/12.svg

```diff
@@ -0,0 +1,5 @@
+    <circle style="fill:#fff;fill-opacity:1;stroke:none;stroke-width:0;stroke-linecap:round;stroke-miterlimit:4;stroke-dasharray:none" cx="60.676369" cy="40.922352" r="1.41111"/>
+</svg>
```

Comment:
Where are you pulling in the MUTCD files from?  Would it be feasible to pull them in with some kind of bulk import or are they scattered through wikimedia commons or what's the situation?

Reply:
> The SVG files were downloaded from Wikimedia Commons (which got them from the original PDF), then tweaked by hand to fit into 32x32 pixels, and then optimized with svgo.

I don't think a bulk import is feasible if you want them to be pixel-aligned like I made them. If not, maybe you could import them from Commons, but a few of those still need cleanup to be made consistent. There might also be a way to automatically get them from the PDF but I haven't looked into it.

Reply:
Ahh I should probably read your comments first 😂.  There's probably no getting around making icons pixel aligned.  Be aware that raster zooms (which osm-carto uses) and vector zooms are not equivalent - it is off by exactly one level.